File: tarea2/4.py
def fecha():
    meses={1:"Enero", 2:"Febrero", 3:"Marzo", 4:"Abril", 5:"Mayo", 6:"Junio", 7:"Julio", 8:"Agosto", 9:"Septiembre", 10:"Octubre", 11:"Noviembre", 12:"Diciembre"}
    input_fecha = input("Ingrese la fecha en formato dd/mm/aaaa: ")
    dia = (input_fecha[0:2])
    mes = int(input_fecha[3:5])
    anio = int(input_fecha[6:10])
    print(f"La fecha ingresada es: {dia} de {meses[mes]} del {anio}")
    
fecha()

File: tarea2/4.java
import java.util.HashMap;
import java.util.Scanner;

public class Fecha {

    public static void main(String[] args) {
        HashMap<Integer, String> meses = new HashMap<>();
        meses.put(1, "Enero");
        meses.put(2, "Febrero");
        meses.put(3, "Marzo");
        meses.put(4, "Abril");
        meses.put(5, "Mayo");
        meses.put(6, "Junio");
        meses.put(7, "Julio");
        meses.put(8, "Agosto");
        meses.put(9, "Septiembre");
        meses.put(10, "Octubre");
        meses.put(11, "Noviembre");
        meses.put(12, "Diciembre");

        Scanner scanner = new Scanner(System.in);

        System.out.print("Ingrese la fecha en formato dd/mm/aaaa: ");
        String inputFecha = scanner.nextLine().trim();

        String[] partesFecha = inputFecha.split("/");
        int dia = Integer.parseInt(partesFecha[0]);
        int mes = Integer.parseInt(partesFecha[1]);
        int anio = Integer.parseInt(partesFecha[2]);

        System.out.printf("La fecha ingresada es: %d de %s del %d\n", dia, meses.get(mes), anio);

        scanner.close();
    }
}
